Automattic/wp-calypso
# PlansGrid: cleanup duplicated component

#### Changes proposed in this Pull Request
* Remove `plans-item` duplicated component introduced at #44582 revert.

#### Testing instructions
* go to [/new](https://calypso.live/new?branch=fix/plans-grid-duplicated-component)
* open plans grid and everything should be as in production
